src/Microsoft.Cci.Extensions/Extensions/AssemblyIdentityHelpers.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Cci.Extensions
+{
+    public static class AssemblyIdentityHelpers
+    {
+        public static string Format(this AssemblyIdentity assemblyIdentity)
+        {
+            var name = new System.Reflection.AssemblyName();
+            name.Name = assemblyIdentity.Name.Value;
+#if !COREFX
+            name.CultureInfo = new CultureInfo(assemblyIdentity.Culture);
```

Comment:
It's definitely possible. I looked through the code using it  and I believe the only things using this particular helper are just using it for string matching / hashing in ApiReviewer and ApiPort. They may just continue to work if all the strings generated by this are still consistent with each other. If we move those tools to CoreCLR, we will have to make sure that any behavior differences caused by this are okay. In the meantime, the original behavior is still available if you don't define COREFX.